RACNest/ViewControllers/Form/FormViewController.swift

```diff
@@ -25,8 +25,8 @@ final class FormViewController: UIViewController {
         
         viewModel.username <~ usernameField.rex_textSignal
         viewModel.password <~ passwordField.rex_textSignal
-        
-        loginButton.rex_pressed <~ SignalProducer(value: viewModel.authenticate)
+
+        loginButton.rex_pressed <~ SignalProducer(value: CocoaAction(viewModel.authenticateAction) { _ in })
         
```

Comment:
I think the VC was already aware of the `CocoaAction` in the first place when he was accessing the `authenticate` property of the VM, the difference here is the place where this initialization is performed. But I can be wrong, I'm still trying to understand where I think that this should be done.